bson/bsonrw/extjson_wrappers.go

```diff
@@ -199,6 +203,9 @@ func (ejv *extJSONValue) parseDateTime() (int64, error) {
 
 func parseDatetimeString(data string) (int64, error) {
 	t, err := time.Parse(rfc3339Milli, data)
+	if err != nil {
+		t, err = time.Parse(rfc3339MilliFallback, data)
```

Comment:
Do you really want to overwrite the error? If the fallback doesn't work, would it be better to report the original error?

Reply:
I considered having a new error variable, but this function always reports an error in the form of "invalid $date value string: <dateString>" so the actual error from `Parse` doesn't matter. We only care whether it's `nil` or not.

Reply:
Ack.